cfy_manager/main.py

```diff
@@ -949,6 +950,39 @@ def wait_for_starter(verbose=False, timeout=180):
     journalctl.kill()
 
 
+def _guess_private_ip():
```

Comment:
this is equivalent to the `ip a | grep | grep | grep` that we had

Reply:
Looks like it, yes. Would it benefit from having expected output structure in a comment?